application-analytics-ui/src/main/resources/Analytics/Code/ApplicationsPanelEntry.xml

```diff
@@ -170,9 +170,11 @@
       <name>platform.panels.AnalyticsApplication</name>
     </property>
     <property>
-      <parameters>label=$services.localization.render('analytics.extension.title')
-        target=Analytics.WebHome
-        icon=icon:chart-bar</parameters>
+      <parameters>#if ($xwiki.hasAccessLevel('view', 'Analytics.WebHome'))
+  label=$services.localization.render('analytics.extension.title')
+          target=Analytics.WebHome
+          icon=icon:chart-bar
+#end</parameters>
```

Comment:
```
<parameters>## If one property is empty then the entry point won't be listed. Thus we check for view rights since we don't want to show the Analytics Application otherwise.
label=#if($xwiki.hasAccessLevel('view', 'Analytics.WebHome'))$services.localization.render('analytics.extension.title')#end
target=Analytics.WebHome
icon=icon:chart-bar
</parameters>
```